fs/copy_test.go

```diff
@@ -6,7 +6,7 @@ import (
 
 	_ "crypto/sha256"
 
-	"github.com/docker/containerd/fs/fstest"
+	"github.com/docker/containerd/testutil/fstest"
```

Comment:
I'm not sure I'm on board with this package name. I really don't like expanding `util` packages. They end up with no clear purpose and just become a dumping ground.